gnmi/gnmistore/datastore.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package gnmistore contains a gNMI cache implementation that can be used as
+// an on-device/fake device implementation.
+package gnmistore
```

Comment:
optional: does this need to be in its own package?

Reply:
I created this because there was a circular dependency. I'm looking to remove either gnmit/gnimstore once the server is cleaned up.